thingif/src/main/java/com/kii/thingif/query/GroupedHistoryStatesQuery.java

```diff
@@ -70,7 +70,7 @@ public Builder setClause(@Nullable QueryClause clause) {
 
         /**
          * Set firmware version for grouped query.
-         * @param firmwareVersion firmware version
+         * @param firmwareVersion older firmware version, which defined this alias.
```

Comment:
>  equals or earlier than a firmware version which defined the states

You can't said the earlier version also defined the alias, it is not true. 

Let me give example: 

```
v1 defined AirConditionerAlias
v2 defined AirConditionerAlias and HumidityAlias
v3 only defined NewAlias
```
we assume a thing is upgraded to v3
if user would like to query states of  AirConditionerAlias, which is not in current version. User needs to provide v2 or v1. And states of AirConditionerAlias in v1 and v2 will response.

if user would like to query states of  HumidityAlias, which is not in current version. User needs to provide v2. Because HumidityAlias is not defined in v1, so only states of HumidityAlias in v2 response.